nodejs/week2/src/backend/index.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express");
+const app = express();
+const port = process.env.PORT || 3000;
+
+const mealsRouter = require("./routes/meals");
```

Comment:
It is not necessity to have both in index.js

```
app.get("/meals", meals);
app.get("/meals/:id", meals);
```
You can use only one specific router fx
```
// in index.js
app.use('/meals', meals);
// in meals router.js
router.get('/', (req, res) => { /* return meals */ }
router.get('/:id', (req, res) => { /* return a meal by id */ }
```